### ISSUE TITLE
Last.fm scrobbing broken and list sorting

### ISSUE THREAD

**Issue by szekeres2016** [Quacky2200/Spotify-Web-Player-for-Linux#15]
Great application but there are a few minor inconveniences on ubuntu 16.04 I wish to note. I think the problem is with the web player not the application itself but still, last.fm account is linked but scrobbing feature seems to be broken. Also my favorite list is sorted with the oldest songs at the top, since I prefer to listen the newer ones I really like away to reverse the sorting so the newly added ones are at the top. If these issues could be looked into I would not hesitate to remove the old desktop client. Thank you for the great work so far!

**Comment by Quacky2200** [comment]
I do not have a Last.FM account to test this - there are quite a lot of bugs at the moment and I'm not to sure this will be at the top of the list

**Comment by Quacky2200** [comment]
I will look into this for the next version

**Comment by szekeres2016** [comment]
Little update on this, I am on the wonderful new 1.0 release of the app and my latest songs were scrobbed to Last.fm. I will test it a bit more just to give an accurate review of this. The playlist sorting thing I could figure it out, I could sort the list the way I want (new tracks on top) inside the Android app, and the changes made there are reflected in the webapp as well, so that is a nice workaround for that! :D Good luck with this project, really excited to see whats next.

**Comment by Quacky2200** [comment]
Sounds like it may have been something Spotify have dealt with themselves? I had never touched Last.fm for all my sync's so far. _It could_ have been some type of bug that went unseen in an older release that was somehow solved during the last few releases but I think that's a _very rare_ possibility

Apart from that I'm glad it's working the way you want, not much is happening right now except stabilisation and typical fixes. I'm always open to feature requests

**Comment by marpala** [comment]
Last.fm is also broken for me on Ubuntu 16.10, but in my case the application crashes as soon as I click on "Connect" [to Last.fm] in the options.

**Comment by Quacky2200** [comment]
Yes, Spotify needs to go to another page and Spotify will hide. The best thing is to do what you did and set it up manually for now
